addons/mail/static/src/models/composer_view/composer_view.js

```diff
@@ -41,6 +47,20 @@ registerModel({
         closeSuggestions() {
             this.update({ suggestionDelimiterPosition: clear() });
         },
+        /**
+         * Returns whether the given html node is inside this composer view,
+         * including whether its inside the emoji popover when active.
```

Comment:
its -> it is

Reply:
also maybe change variable and doc name from node to element according to type change?